configure

```diff
@@ -81,6 +83,12 @@ try_cxx_argument()
     fi
 }
 
+# Same as try_cxx_argument but doesn't return 1 on unsupported flag
+try_optional_cxx_argument()
+{
+    try_cxx_argument $1 $2 || :
```

Comment:
These (`$1` and `$2`) should really be quoted (`"$1" "$2"`) or they are prone to word-splitting. It's probably not a problem with current uses, but it's best practice.